### ISSUE TITLE
Loss=nan when finetune the model with autocast()

### ISSUE THREAD

**Issue by trinhvg** [mahmoodlab/UNI#59]
I can finetune without autocast  or linprobe the model but got loss = nan if i finetune with autocast.
Anyone have any suggestion to fix this?

        with torch.cuda.amp.autocast():
            output = model(images)
